src/argilla/client/feedback/dataset/mixins.py

```diff
@@ -379,3 +382,380 @@ def unify_responses(
+        )
+
+    @classmethod
+    def for_summarization(cls: Type["FeedbackDataset"], use_markdown: bool = False) -> "FeedbackDataset":
+        """
+        You can use this method to create a basic dataset for summarization tasks.
+        To add items to your dataset, use the "add_item" method.
+
+        Args:
+            None
+
+        Returns:
+            A `FeedbackDataset` object for summarization containing "text" field and a TextQuestion named "summary"
+        """
+        return cls(
+            fields=[
+                TextField(name="text", use_markdown=use_markdown),
+            ],
+            questions=[
+                TextQuestion(name="summary", description="Write a summary of the text.", use_markdown=use_markdown)
+            ],
+            guidelines="This is a summarization dataset that contains texts. Please summarize the text in the text field.",
+        )
+
+    @classmethod
+    def for_translation(cls: Type["FeedbackDataset"], use_markdown: bool = False) -> "FeedbackDataset":
+        """
+        You can use this method to create a basic dataset for translation tasks.
+        To add items to your dataset, use the "add_item" method.
+
+        Args:
+            labels: Set this parameter to True if you want to add labels to your dataset
+
+        Returns:
+            A `FeedbackDataset` object for translation containing "text" field and a TextQuestion named "translation"
+        """
+        return cls(
+            fields=[TextField(name="text", use_markdown=use_markdown)],
+            questions=[TextQuestion(name="translation", description="Translate the text.", use_markdown=use_markdown)],
+            guidelines="This is a translation dataset that contains texts. Please translate the text in the text field.",
+        )
+
+    @classmethod
+    def for_supervised_fine_tuning(
+        cls: Type["FeedbackDataset"], context: bool = True, use_markdown: bool = False
+    ) -> "FeedbackDataset":
+        """
+        You can use this method to create a basic dataset for supervised fine-tuning tasks.
+        To add items to your dataset, use the "add_item" method.
+
+        Args:
+            context: Set this parameter to True if you want to add context to your dataset
+            use_markdown: Set this parameter to True if you want to use markdown in your dataset
+
+        Returns:
+            A `FeedbackDataset` object for supervised fine-tuning containing "instruction" and optional "context" field and a TextQuestion named "response"
+        """
+        return cls(
+            fields=[
+                TextField(name="prompt", use_markdown=use_markdown),
+                TextField(name="context", use_markdown=use_markdown),
```

Comment:
this should be `TextField(required=False)` because we might want a context field but not always use it